great-idea-website/index.html

```diff
@@ -14,50 +14,102 @@
+        </p>
+    </div>
+  </section>
 
-  Services
-  Product
-  Vision
-  Features
-  About
-  Contact
-
-  <img class="logo" src="img/logo.png" alt="Great Idea! Company logo.">
-  Innovation
-  On
-  Demand
-
-  Get Started
-
-  <img src="img/header-img.png" alt="Image of a code snippet.">
-
-  Features
-  Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
-
-
-  About
-  Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
-
-  <img class="middle-img" src="img/mid-page-accent.jpg" alt="Image of code snippets across the screen">
   
-  Services
-  Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
-
-
-  Product
-  Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
-
-  Vision
-  Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
-
-  Contact
-  123 Way 456 Street
-  Somewhere, USA
-  1 (888) 888-8888
-  sales@greatidea.io
-
-  Copyright Great Idea! 2018
+  <section class="section3">
+    <div class="main3">
+  <img class="middle-img" src="img/mid-page-accent.jpg" alt="Image of code snippets across the screen">
+    </div>
+  </section>
+
+  <div class="Section4">
+    <span class="Services">
+    <h5>Services</h5>
+      <p>Aliquam elementum magna eros, ac posuere elvit tempus et. 
+        Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu 
+        ornare nisl. Nullam convallis augue justo, at imperdiet metus 
+        scelerisque quis.
+      </span>
+      </p>
+
+    <span class="Products">
+    <h5>Product</h5>
+      <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse 
+      vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
+      </p>
+    </span>  
   
-  </body>
+    <span class="Vision"> 
+    <h5>Vision</h5>  
+      <p>Aliquam elementum magna eros, ac posuere elvit tempus et. 
+      Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam 
+      convallis augue justo, at imperdiet metus scelerisque quis.</p>
+      </span>
+    
+  </div>
+    
+<section class="contact"> 
+    <div class="bottomMain">
+    <h4>Contact</h4>
+      <address>123 Way 456 Street
+     <br>Somewhere, USA
```

Comment:
Try to indent child elements so that you can see the nice hierarchy of your html tree 🌲 It's most common to see either 2 spaces or 1 tab (4 spaces) for indentation and you should stick to one :)
```suggestion
     <div class="bottomMain>
       <h4>Contact</h4>
       <address>
         123 Way 456 Street<br>
         Somewhere, USA<br>
         1 (888) 888-8888<br>
         sales@greatidea.io
       </address>
     </div>
```